Makefile

```diff
@@ -132,7 +135,7 @@ endif
 
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd | kubectl apply -f - || true
```

Comment:
why failed apply is ignored?

Reply:
Currently, build service does not bring any new CRDs, so build for empty dir `config/crd` returns nothing. To avoid error for the make target, the ignore was added.
I decided to keep `install` command and `crd` directory to not to break common pattern. But if you think that it's better to remove them - let me know.